@stellar/design-system/src/components/Button/styles.scss

```diff
@@ -59,7 +59,9 @@
   background-color: var(--Button-color-background-default);
   border-radius: pxToRem(4px);
   cursor: pointer;
-  transition: all var(--anim-transition-default);
+  transition: color var(--anim-transition-default),
```

Comment:
extra space?

Reply:
It looks OK in my code editor. Thanks for paying attention to little things like this! :raised_hands: 

![image](https://user-images.githubusercontent.com/7346473/221036602-4f59a239-ef0c-4154-a40a-1b80051f99b4.png)

Reply:
oh whoops, didn't notice those following lines are also part of the transition.